dev/ledger.md

```diff
@@ -751,6 +857,47 @@ An asset freeze transaction additionally has the following fields:
 The cryptographic hash of the fields above is called the _transaction
 identifier_.  This is written as $\Hash(\Tx)$.
 
+## State proof transaction
+
+A special transaction is used to disseminate and store state 
+proofs.  The type of a state proof transaction is `stpf`.
+This type of transaction must always be issued from a special sender
```

Comment:
What about first valid/last valid rounds? Are there any constraints?

Reply:
actually, we use the first valid param to calculate the target proven weight. please take a second look at the signed weight constraints section.